werf.yaml

```
@@ -120,12 +120,15 @@ shell:
       cat images_digests.json
 ---
 image: python-dependencies
-from: python:3.9-slim
+from: {{ .Images.BASE_ALT_P11 }}
 fromCacheVersion: "2024-11-07.1"
 git:
   - add: /lib/python/requirements.txt
     to: /requirements.txt
```

```suggestion
git:
  - add: /lib/python/requirements.txt
    to: /requirements.txt
    stageDependencies:
      beforeInstall:
        - '**/*'
```
For this case, it's better to use `stageDependencies` rager than `fromCacheVersion`

---

```
Error: unable to generate default repo: unable to load werf config: unknown fields: `beforeInstall`!

    beforeInstall:
    - '**/*'

/tmp/werf-config-render-3021092801
```